packages/modal/src/Modal.styles.ts

```diff
@@ -0,0 +1,146 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies } from '@leafygreen-ui/tokens';
+import facepaint from 'facepaint';
+import { transparentize } from 'polished';
+import { CloseIconColor, ModalSize } from './types';
+
+// breakpoints for different screen sizes
+export const small = '767px'; // mobile screens, from 0px - 767px
+export const medium = '768px'; // tablet screens, from 768px - 1024px
+export const large = '1025px'; // laptops/desktop screens, from 1025px and above
```

Comment:
Unrelated to the PR changes, but we should import breakpoints from our tokens library

Reply:
Leaving for now because these don't match the breakpoints from the tokens library

Reply:
Could run this by design quickly to see if we're ok to move to the system's breakpoints. Not sure if that makes this a breaking change, though.

Reply:
yeah I remember Sooa was working on updating Modal widths, i'll check in with her

Reply:
Really looking at this, i don't even think we need this package. This is targeting 3 different breakpoints but the styles are only change at the `large` breakpoint. I think i'm gonna remove it.

Reply:
sgtm